source/client/output_formatter_impl.cc

```diff
@@ -142,15 +145,25 @@ std::string ConsoleOutputFormatterImpl::statIdtoFriendlyStatName(absl::string_vi
   return std::string(stat_id);
 }
 
-std::string JsonOutputFormatterImpl::formatProto(const nighthawk::client::Output& output) const {
-  return Envoy::MessageUtil::getJsonStringFromMessageOrDie(output, true, true);
+absl::StatusOr<std::string>
+JsonOutputFormatterImpl::formatProto(const nighthawk::client::Output& output) const {
+  auto status_proto = Envoy::MessageUtil::getJsonStringFromMessage(output, true, true);
+  if (status_proto.ok()) {
+    return status_proto.value();
+  } else {
```

Comment:
Consider the ternary operator here, less code, less impact on coverage for the lines we can't cover. And personally I like that it makes it slightly more obvious that the method inescapable returns a value at a glance.

Reply:
if the `else` part of this code doesn't ever execute, how is the idea of simply returning `status_proto.value()`?
We can get rid of the if-else branching.

Reply:
Hmm, yes. In fact, I think we can just return `status_proto`? E.g. condense the whole method body to a one-liner, `Envoy::MessageUtil::getJsonStringFromMessage(output, true, true);`